### ISSUE TITLE
Investigate addition of median home value threshold to the higher ed formula

### ISSUE THREAD

**Issue by BethMattern** [usds/justice40-tool#1269]
The higher ed threshold may exclude some census tracts that would otherwise be considered an EJ community. Investigate the impact of including median home value in the formula - something like:

Higher ed enrollment is at or above 20% AND median home value is greater than X

X should be determined based on this analysis

**Comment by emma-nechamkin** [comment]
Starting this off with some comments -- this is not yet ready for review. 

I'm hesitant to use MHV in this way. I did a quick analysis, looking at if we restricted median home value to be less than every quantile (so in the 10th or lower, 20th or lower and so on). To get a population that seems (based on the comparison tool) to have a similar profile to Def M, we have to cut this threshold quite low (10ish or 20ish). This adds back very few tracts, and maybe not the tracts I'd have thought of. In other words, I don't think this looks great (but could be convinced otherwise).  I can clean up this analysis, but this conversation might be easier in person @BethMattern @lucasmbrown-usds 

I'm wondering if we could do something like: 20% of students, and no neighbors are DACs. This is similar in how we might think about donut hole DACs -- areas with lots of DACs become difficult to resolve for the purposes of allocating benefits AND reflect systemic vulnerability -- but different in the sense that we are allowing there to be clusters of non-DACs. The idea would be -- yes, there are students, but there's also widespread environmental impacts. 

This would include a tract like: https://screeningtool.geoplatform.gov/en/cejst#15.17/33.750517/-84.41107, but not a tract like: https://screeningtool.geoplatform.gov/en/cejst#15.53/37.877445/-122.260258. I plan to do some more analysis here tomorrow. 

Here, there's some nuance additionally -- we would have to calculate this once simultaneously for all tracts, so if there are clusters of tracts that would have been included, the middle ones might get left out -- but I think it might make more sense than trying to play with another threshold.
